cea/interfaces/dashboard/api/project.py

```diff
@@ -117,57 +114,52 @@ def post(self):
         locator = cea.inputlocator.InputLocator(new_scenario_path)
 
         # Run database_initializer to copy databases to input
-        if 'databases-path' in payload:
+        databases_path = api.payload.get('databases-path')
+        if databases_path is not None:
             try:
-                cea.api.data_initializer(config, scenario=new_scenario_path, databases_path=payload['databases-path'])
+                cea.api.data_initializer(config, scenario=new_scenario_path, databases_path=databases_path)
             except Exception as e:
                 trace = traceback.format_exc()
                 return {'message': 'data_initializer: {}'.format(e.message), 'trace': trace}, 500
 
-        if payload['input-data'] == 'import':
-            files = payload['files']
-
+        input_data = api.payload.get('input-data')
+        if input_data == 'import':
+            files = api.payload.get('files')
             if files is not None:
                 try:
-                    # Local variables
-                    if 'terrain' not in files:
-                        files['terrain'] = ''
-                    if 'streets' not in files:
-                        files['streets'] = ''
-                    if 'surroundings' not in files:
-                        files['surroundings'] = ''
-                    if 'typology' not in files:
-                        files['typology'] = ''
                     cea.api.create_new_scenario(config,
-                                               scenario=new_scenario_path,
-                                               zone=files['zone'],
-                                               surroundings=files['surroundings'],
-                                               streets=files['streets'],
-                                               terrain=files['terrain'],
-                                               typology=files['typology'])
-
+                                                scenario=new_scenario_path,
+                                                zone=files.get('zone'),
+                                                surroundings=files.get('surroundings', ''),
```

Comment:
just out of curiosity... could the `''` also be `None`? I don't remember anymore, but I think that would be the "cleaner" version, no?

Reply:
Currently **`PathParameter`** is unable to encode `None`, it would just cast it to a **`str`**. And `create_new_scenario` checks if it is an empty string for the path parameters as it decodes to a **`str`**. 

We could override the encode method to parse `None` to an empty string.